src/pages/Home/HomeController.ts

```diff
@@ -0,0 +1,35 @@
+import { navigate } from 'utils/helpers';
+
+import HomeModel from './HomeModel';
+import HomeView from './HomeView';
+
+export default class HomeController {
+  model: HomeModel;
+  view: HomeView;
+
+  constructor(model: HomeModel, view: HomeView) {
+    this.model = model;
+    this.view = view;
+    this.init();
+  }
+
+  async init(): Promise<void> {
+    const mainGifData = await this.model.toJson();
+    const gifUrl = mainGifData.original.url;
+    this.view.render(gifUrl as string);
```

Comment:
gifUrl이 없을때 그리지 않거나 또는 render에서 gifUrl타입을 string | undefined로 변경해야해요

```suggestion
    if (gifUrl) {
            this.view.render(gifUrl);
    }
```